vllm/_custom_ops.py

```diff
@@ -532,6 +532,52 @@ def cutlass_scaled_mm_azp(a: torch.Tensor,
     return out
 
 
+def cutlass_compress_entry(a: torch.Tensor) \
+    -> Tuple[torch.Tensor, torch.Tensor]:
+    assert (a.dtype is torch.int8 or a.dtype is torch.float8_e4m3fn or \
```

Comment:
nit: could be `assert a.dtype in [torch.int8, torch.float8_e4m3fn, torch.bfloat16, torch.float16]`